generator/lib/ArtifactProviders/ExecutableSeed/babel-plugin-transform-template.ts

```diff
@@ -61,6 +67,7 @@ export type BabelPluginTransformTemplateOptions = {
     content: string
     path: string
     datasourceUrlEnvironmentVariableName: string
+    transformedPrismaSchemaContentEnvironmentVariableName: string
```

Comment:
Passing the transformed schema contents in like this is not strictly necessary - we could even use the original schema contents now that PCW is gone. However, it's easier to do it like this because the multiline content of is hard to format.